### PR TITLE
Beacon screen UI

### PR DESCRIPTION
**Pull request by joriba**
I added a beacon screen about how I envision it. 
Since there is no proper navigation yet (Issue #133), the way to test is via the preview or by faking the navigation.
I think it looks quite nice, let me know what you think :)

The idea behind the google map is to show on a map where the beacon is (WIP). Maybe in the future, this can be replaced by information about the beacon's creator or something.

### CODE REVIEW

**Review by mbouaissi — APPROVED** [review]
It looks very good! I really like the fact that we can see the map (which will be i guess center to the beacon), and also I really like how the title are displayed. I don't know if it's hard to do, but maybe we could have on the card the profile picture of the user who has drop the title, and when we click on it, we can see his profile (but i think it can clearly be a issue on itself, we could do it for the next sprint). But for now it's really good

**Comment by joriba** [comment]
> It looks very good! I really like the fact that we can see the map (which will be i guess center to the beacon), and also I really like how the title are displayed. I don't know if it's hard to do, but maybe we could have on the card the profile picture of the user who has drop the title, and when we click on it, we can see his profile (but i think it can clearly be a issue on itself, we could do it for the next sprint). But for now it's really good

I agree that this will be nice (although maybe I'd display it somewhere else thanon the map). But it definitely is an issue for later, I don't think we even have the users linked to the beacons yet, nor do they have profile pictures.

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt [hunk 1]
```diff
@@ -32,58 +54,120 @@ fun BeaconScreen(
...
+  HorizontalDivider()
+  Text(
+      text = stringResource(R.string.beaconTracksTitle),
+      style = MaterialTheme.typography.displayMedium,
+      modifier = Modifier.testTag("beaconTracksTitle"))
+  LazyColumn { items(beacon.tracks) { TrackItem(it) } }
+}
+
+@Composable
+internal fun TrackItem(track: Track) {
```
Comment (KlaKalma) [inline app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt:139]: fancy this one haha, internal

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/LoginScreen.kt
[the remark this answers is not in the document]
Comment (joriba) [reply app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/LoginScreen.kt:1]: Oh yeah, I forgot to take it out before commiting, I'll fix

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/App.kt [hunk 2]
```diff
@@ -73,7 +73,7 @@ fun AppScaffold(navController: NavHostController) {
         SurroundWithMiniPlayer(showBottomBar) {
           NavHost(
               navController = navController,
-              startDestination = Route.BEACON.routeString,
+              startDestination = Route.LOGIN.routeString,
```
[the remark this answers is not in the document]
Comment (joriba) [reply app/src/main/java/ch/epfl/cs311/wanderwave/ui/App.kt:76]: What do you mean? I changed `startDestination = Route.BEACON.routeString` to `LOGIN.routeString`, so it starts at the login again.

[the remark this answers is not in the document]
Comment (KlaKalma) [reply app/src/main/java/ch/epfl/cs311/wanderwave/ui/App.kt:76]: I'm sorry I wasn't clear, I was talking about what I did, commenting out the login thing and having the route like that, is this a good way of doing? You just put back what I changed.

**Pull request merged**